docs/logging/docs/010-architecture-logging.md

```diff
@@ -2,8 +2,19 @@
 title: Architecture
 ---
 
-This document outlines the logging architecture of Kyma. It highlights information sources from which Logspout extracts logs to feed to OK Log.
+This document outlines the logging architecture of Kyma. It highlights information sources from which Promtail extracts logs to feed to Loki.
 
-Logspout is deployed as a stateless Daemonset and shares `/var/run/docker.sock` from the node. It then feeds the logs to OK Log through the ingest API.
+#### Agent (Promtail)
+Agent is responsible for obtaining reliable metadata that is consistent with the metadata associated with the time series / metrics. To achieve this agent use the same service discovery and label relabelling libraries as Prometheus. Agent packed up in a daemon that discovers targets, produces metadata labels and tails log files to produce stream of logs, which will be buffered on client side and the sent to the service.
```

Comment:
Isn't that mainly copied from the original documentation? I suggest to just link the docu instead of copying it. Focus here on the Kyma specific parts like the setup. What kind of persistence is used, what are the configuration options and where to apply them

Reply:
Link to original design doc and example storage configuration for GCS added.